### ISSUE TITLE
extension does not read the configuration file

### ISSUE THREAD

**Issue by wehrstedt** [otris/vscode-janus-debug#121]
The extension does not read the existing launch.json.

On the first start the extension ask me for the credentials even a launch.json exists in the project.
When I type in my credentials and save the new configuration, my existing configuration file will be overwritten by the extension.

Version: 0.0.17

**Comment by bkircher** [comment]
I presume the action you do before credentials are asked is upload or download a scripts from a DOCUMENTS server, right?

**Comment by tineheller** [comment]
Well no... In the last weeks I inserted the flag `currentConfiguration` in `launch.json`.
If that flag is not set in your `launch.json` the extension couldn't find your credentials. I changed that. Now you only need to set the flag, if you use more than one configuration.

I did some more changes. Unfortunately with that changes the `launch.json` will be overwritten. I don't like that! So I fixed it and I will publish it immediately.
